releases/release-1.21/release-notes-draft.md

```diff
@@ -1 +0,0 @@
-## [Moved to be under `release-notes/` directory](release-notes/release-notes-draft.md)
```

Comment:
oh let's bring this file back 😄 something like

```sh
git checkout master -- releases/release-1.21/release-notes-draft.md
```

Reply:
to clarify -- this file should not show up at all in `git diff master`

Reply:
Restored it.